### PR TITLE
Uploader: allow to add more files via drop

### LINKED ISSUE #5285: [v4] Uploader window should allow addition of new files via drag and drop
## Description
In the new uploader window, I'd expect to be allowed to drag new files into the window until the upload button is clicked and upload all my files at once.
CleanShot 2023-06-14 at 10 27 56@2x

This is especially useful when you have e.g. files from different folders and want to upload them at once. 

## Your setup
**Kirby Version**  4.0-alpha.2

### PR DESCRIPTION
**Pull request by distantnative**
### Enhancement
- Uploader dialog allows to add additional files via drag and drop 
#5285

### CODE REVIEW

File: panel/src/panel/upload.js [hunk 1]
```diff
@@ -167,11 +167,15 @@ export default (panel) => {
 			files = [...files];
 
 			// add all files to the list as enriched objects
-			this.files = files.map((file) => this.file(file));
+			files = files.map((file) => this.file(file));
+
+			// merge the new files with already selected files
+			this.files = [...this.files, ...files];
```
Comment (bastianallgeier) [inline panel/src/panel/upload.js:173]: I think we somehow need to detect duplicates here.

Comment (distantnative) [reply panel/src/panel/upload.js:173]: What would you think is a good way to detect this? Comparing the `src` property?

Screenshot 2023-06-19 at 19 44 03

Comment (bastianallgeier) [reply panel/src/panel/upload.js:173]: I guess we could only combine lastModified + name + size.

Comment (bastianallgeier) [reply panel/src/panel/upload.js:173]: LOL. Notice to myself – First step: check out changes. Second step: leave comment 🙈

**Review by bastianallgeier — APPROVED** [review]

**Pull request merged**